.github/workflows/tests.yml

```diff
@@ -264,18 +264,18 @@ jobs:
        run: |
          ./.ci.sh -t
 
-  macos-12:
-    name: macOS 12.0 (Monterey)
+  macos-14:
+    name: macOS 14.0 (Sonoma)
```

Comment:
Given that the macOS 14 image is the version that runs on M1 Macs, is it possible that we want to promote this to one of the actions that runs on PRs?

Reply:
Sure! I expect macOS14 to pick up either installation failures (which don't have to run on every PR) or numerical stability failures (which would be better to have on every PR). For the latter reason, we may also want to have results of `batch_processing.sh` for both macOS 12 and macOS 14 in the next release.